### DISCUSSION TITLE
Add a map plugin

### DISCUSSION THREAD

**Discussion by raphaelm** [pretix/pretix#1939] [Ideas]
* Add an event geo-location field to pretix core
* Add a plugin that displays a Google/OSM map to the front page of an event

**Comment by oocf** [comment]
I think that this plugin will be better to be handle by event, each event could have its map to see where the event is going to be, rather than in the principal page of the ticket shop

**Comment by raphaelm** [comment]
@oocf 	I agree, and I even meant that. You're right, I wrote it in a way that could easily be misunderstood

**Comment by TimmThaler** [comment]
In the backend at "Settings" > "Tickets" > "Download Formats" there are coordinates for the use with passbook tickets. Perhaps it would be possible to use these coordinates to show an iframe / js script of openstreetmaps for each event under its header next to the address.

https://wiki.openstreetmap.org/wiki/DE:Karte_in_Webseite_einbinden
